modules/consentManagement.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * This module adds GDPR consentManagement support to prebid.js.  It interacts with
+ * supported CMPs (Consent Management Platforms) to grab the user's consent information
+ * and make it available for any GDPR supported adapters to read/pass this information to
+ * their system.
+ */
+import * as utils from 'src/utils';
+import { config } from 'src/config';
+import { setTimeout } from 'core-js/library/web/timers';
```

Comment:
What's the purpose of this?  All the browsers we support seem to support `setTimeout` in the way you are using it w/o this polyfill.

Reply:
I removed it, it got added automatically.